sample_apps/run_example.sh

```diff
@@ -47,6 +49,10 @@ Jq=jq    # Needed for OE app; will be established later on.
 MbedTLS_dir="mbedtls"
 Simple_App_under_gramine_MbedTLS_dir="sample_apps/simple_app_under_gramine/${MbedTLS_dir}"
 
+# Work-instructions for SEV-app need following files pre-populated by the user.
+#
+Platform_data_files_SEV="sev_policy.json ark_cert.der ask_cert.der vcek_cert.der"
```

Comment:
NOTE: The work instructions need to be updated to mention `platform_data` and these files

Reply:
Done. Added a note to instructions.md about these 4 files.